static/src/stylesheets/module/experiments/_remind-me.scss

```diff
@@ -0,0 +1,41 @@
+$remind-me-color: $features-support-1;
+.remind-me {
+    background-color: $remind-me-color;
+    padding: $gs-baseline/4 $gs-gutter/2 $gs-baseline/2;
+    border-top: 3px solid $features-support-3;
+    color: #ffffff;
+}
+
+.remind-me__subtitle {
+    @include fs-textSans(5);
+    font-weight: bold;
```

Comment:
No, I think this is ok actually - it's applied to here 
![image](https://cloud.githubusercontent.com/assets/638051/17433620/9f696506-5afd-11e6-83d2-255caf270dd0.png) but I thought 'subtitle' would be a heading and using the heading mixins but its not, so nevermind!